tests/auto_inject/test_auto_inject_install.py

```diff
@@ -0,0 +1,248 @@
+import requests
+import os
+from utils import scenarios, features
+from utils.tools import logger
+from utils.onboarding.weblog_interface import make_get_request, warmup_weblog
+from utils.onboarding.backend_interface import wait_backend_trace_id
+from utils.onboarding.wait_for_tcp_port import wait_for_port
+from utils import irrelevant
+from utils import scenarios, context, features
+from utils.virtual_machine.vm_logger import vm_logger
+import pytest
+
+
+class _AutoInjectBaseTest:
+    @irrelevant(
```

Comment:
What's the point of flagging a non-test method with this decorator? If I'm not wrong, there are no effect ?

Reply:
upppsss you are right it's a mistake. This method was a test method, but I finally I renamed but I didn't remove the decorator